keylime/registrar_common.py

```diff
@@ -212,6 +212,36 @@ def do_GET(self) -> None:
 
         web_util.echo_json_response(self, 200, "Success", version_info)
 
+    @staticmethod
+    def get_network_params(
+        json_body: Dict[str, Any], agent_id: str
+    ) -> Tuple[Optional[int], Optional[int], Optional[str]]:
```

Comment:
The IP address should be of type `str`, shouldn't?

Reply:
Correct. Thanks. Fixed.